### DISCUSSION TITLE
insert with vector of uuids as a column?

### DISCUSSION THREAD

**Discussion by digevgreg** [SeaQL/sea-query#488] [Q&A] [answered]
Hi,

I'm trying to do a very basic Insert.  It works perfectly if I only insert "Id" and "Owner" which are both of type uuid (from the uuid crate).  So these support the "into()" trait so it just converts these uuid values to whatever values_panic wants (I guess?).

The problem is with my "categories" column which is an array of UUID in Postgres and a Vec<Uuid> in Rust.  So how do I convert this Vec<Uuid> into whatever "values_panic" is looking for?  Which according to the documentation takes a: IntoIterator<Item = SimpleExpr>

Thank you!

        let (sql, values) = Query::insert()
        .into_table(MyTable::Table)
        .columns([
            MyTable::Id,
            MyTable::Owner,
            MyTable::Categories,
        ])
        .values_panic(
            manager.unwrap().into(),
            owner.unwrap().into(),
            ???categories????
    )
        .returning_col(MyTable::Id)
        .build_sqlx(PostgresQueryBuilder);

**Comment by billy1624** [comment, marked as answer]
Hey @digevgreg, welcome! A sanity check: are you on SeaQuery 0.27 and enabled `postgres-array` feature?

**Reply by digevgreg** [reply]
Cargo.toml looks like:

sea-query = { version = "^0", features = ["derive"] }
sea-query-binder = { version = "^0", features = ["sqlx-postgres","with-uuid","with-json","runtime-tokio-rustls"] }

**Reply by digevgreg** [reply]
So I guess probably the newest version but no postgres-array

**Reply by digevgreg** [reply]
Adding postgres-array to sea-query-binder seems to have solved my problem!  Thank you Billy!
